apps/web/src/composables/ledger.ts

```diff
@@ -1,26 +1,72 @@
 import SpeculosHttpTransport from '@casimir/hw-transport-speculos'
-import TransportWebHID from '@ledgerhq/hw-transport-webhid'
+import TransportWebUSB from '@ledgerhq/hw-transport-webusb'
 import Eth from '@ledgerhq/hw-app-eth'
+import { ethers } from 'ethers'
 
 export default function useLedger() {
+  const bip32Path = "44'/60'/0'/0/0"
 
-    const bip32Path = '44\'/60\'/0\'/0/0'
+  async function getLedgerEthSigner() {
+    const transport = await _getLedgerTransport()
+    return new Eth(transport)
+  }
 
-    async function getLedgerEthSigner() {
-        const transport = await _getLedgerTransport()
-        return new Eth(transport)
+  async function _getLedgerTransport() {
+    if (import.meta.env.PUBLIC_SPECULOS_PORT) {
+      return await SpeculosHttpTransport.open(
+        `http://127.0.0.1:${import.meta.env.PUBLIC_SPECULOS_PORT}`
+      )
+    } else {
+      return await TransportWebUSB.create()
     }
+  }
 
-    async function _getLedgerTransport() {
-        if (import.meta.env.PUBLIC_SPECULOS_PORT) {
-            return await SpeculosHttpTransport.open(`http://127.0.0.1:${import.meta.env.PUBLIC_SPECULOS_PORT}`)
-        } else {
-            return await TransportWebHID.create()
-        }
-    }
+  async function sendLedgerTransaction(transaction: any) {
+    // TODO: Replace according to selected testnet
+    const infuraProvider = new ethers.providers.JsonRpcProvider(
```

Comment:
Refactor to use import.meta.env.ETHEREUM_RPC variable provided (will add to scripts in a commit here). Will be local 8545 for now, then `dev` env will use a testnet and `master` releases will use mainnet.

Reply:
Going to pull down your recent changes after you push and finish this from the script side. After that, will also address the gas issue you ran into with sending the transaction.

Reply:
@shanejearley - Sounds good. Won't resolve.